src/containers/EventListener/EventListener.js

```diff
@@ -88,137 +60,86 @@ export /* istanbul ignore next */ class EventListenerContainer extends Component
     }
   }
 
-  fetchData() {
-    const { match } = this.props;
-    const { namespace, eventListenerName } = match.params;
-    this.props.fetchEventListener({ name: eventListenerName, namespace });
+  getAdditionalMetadata() {
+    const { eventListener, intl } = this.props;
+
+    if (!eventListener) {
+      return null;
+    }
+
+    const { serviceAccountName, serviceType } = eventListener.spec;
+    return (
+      <>
+        {serviceAccountName && (
+          <p>
+            <span>
+              {intl.formatMessage({
+                id: 'dashboard.eventListener.serviceAccount',
+                defaultMessage: 'ServiceAccount:'
+              })}
+            </span>
+            {serviceAccountName}
+          </p>
+        )}
+        {serviceType && (
+          <p>
+            <span>
+              {intl.formatMessage({
+                id: 'dashboard.eventListener.serviceType',
+                defaultMessage: 'Service Type:'
+              })}
+            </span>
+            {serviceType}
+          </p>
+        )}
+      </>
+    );
   }
 
-  render() {
-    const { intl, error, match, eventListener } = this.props;
+  getTriggersContent() {
+    const { eventListener } = this.props;
 
-    if (error) {
-      return EventListenerContainer.notification({
-        kind: 'error',
-        intl
-      });
-    }
     if (!eventListener) {
-      return EventListenerContainer.notification({
-        kind: 'error',
-        message: intl.formatMessage({
-          id: 'dashboard.eventListener.unavailable',
-          defaultMessage: 'EventListener not available'
-        }),
-        intl
-      });
+      return null;
```

Comment:
Should an error be getting passed to ResourceDetails in this case rather than returning null?

Reply:
That's already handled by ResourceDetails, if either `error` is passed or `resource` (`eventListener` in this case) is falsy it treats it as an error.